nengo_loihi/loihi_cx.py

```diff
@@ -444,6 +453,16 @@ def axon_slots(self):
         """The total number of axonCfg slots used by all axons."""
         return self.slots_per_axon * self.n_axons
 
+    def map_cx_spikes(self, cx_idxs):
+        axon_idxs = (self.cx_to_axon_map[cx_idxs]
```

Comment:
The logic for these three assignments is hard to read IMO ... is it ever the case that one of the three member variables will be defined, but not the other two? If it's all or none, then I think one if statement would be more clear than three if statements (one per variable).

Reply:
Somewhere down the line, I get rid of `axon_to_synapse_map` since we don't actually need it (essentially, `loihi_interface`/NxSDK figure this out themselves). `cx_to_axon_map` and `cx_atoms` are fairly independent. There are many situations where you might define `cx_to_axon_map`, to map different compartments to different axons, but they're not population axons so you don't use `cx_atoms`. Defining `cx_atoms` by itself would be much less common, because typically you're using population axons so that you can have multiple `cx`s running through the same axons, but in theory it's possible.